index.html

```diff
@@ -13,7 +14,85 @@
   <![endif]-->
 </head>
 
-<body>
-  <h1>My Resume</h1>
-</body>
+<div class="container">
+
+  <body>
+    <nav>
+      <nav class="navbar">
+        <a href="#">Home</a>
```

Comment:
Careful! You actually nested all of your code inside of a nav on this assignment. See above -- you opened two different nav divs, but never closed the second one.